src/main/java/no/unit/nva/bare/AuthorityConverter.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.bare;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AuthorityConverter {
+
+
+    public static final String NUM_FOUND_TAG = "numFound";
+    public static final String RESULTS_TAG = "results";
+    public static final String MARCDATA_TAG = "marcdata";
+    public static final String MARC_100 = "100";
+    public static final String MARC_TAG_TAG = "tag";
+    public static final String MARC_SUBFIELDS_TAG = "subfields";
+    public static final String MARC_SUBCODE_TAG = "subcode";
+    public static final String MARC_VALUE_TAG = "value";
+    public static final String MARC_A = "a";
+    public static final String MARC_D = "d";
+    public static final String IDENTIFIERS_MAP_KEY = "identifiersMap";
+    public static final String SCN_KEY = "scn";
+    public static final String FEIDE_KEY = "feide";
+    public static final String ORCID_KEY = "orcid";
+
+    protected List<Authority> getAuthoritiesFrom(JsonObject jsonObject) {
+        int numFound = jsonObject.get(NUM_FOUND_TAG).getAsInt();
+        List<Authority> authorityList = new ArrayList<>(numFound);
+        final JsonArray results = jsonObject.get(RESULTS_TAG).getAsJsonArray();
+        for (JsonElement jsonElement : results) {
```

Comment:
I would probably replace this code with GSON::toJson with custom type adapter

Reply:
Or in fact, a java lambda

Reply:
implemented new parsing routine with help of Gson (plus some new objects) and java streaming api